### PR TITLE
fix: Upgrade minio-operator

### PR DESCRIPTION
**Pull request by branden**
https://jira.d2iq.com/browse/D2IQ-87980

This upgrades minio-operator to bring it into compatibility with the recently bumped MinIO images.

This also updates Loki services to make use of the Tenant CR's support for declaring buckets.

Related kommander test update at https://github.com/mesosphere/kommander/pull/1695.

### CODE REVIEW

**Review by gracedo — APPROVED** [review]

**Comment by cwyl02** [comment]
can we also bump the kubectl image in `services/minio-operator/4.4.16/minio-operator-pre-upgrade/delete-minio-operator-deployment.yaml` ? kubectl:1.21.3 has 4 CVEs

File: services/project-grafana-loki/0.33.2/kustomization.yaml [hunk 1]
```diff
@@ -4,4 +4,3 @@ kind: Kustomization
 resources:
   - project-grafana-loki.yaml
   - minio.yaml
-  - jobs.yaml
```
Comment (cwyl02) [inline services/project-grafana-loki/0.33.2/kustomization.yaml:7]: 👍

**Comment by gracedo** [comment]
> can we also bump the kubectl image in `services/minio-operator/4.4.16/minio-operator-pre-upgrade/delete-minio-operator-deployment.yaml` ? `kubectl:1.21.3` has 4 CVEs

@cwyl02 let's handle that in a follow up PR! We can bump all the various kubectl images together if there are more than one to bump

**Review by cwyl02 — APPROVED** [review]

**Comment by cbuto** [comment]
@mesosphere-mergebot backport release-2.2

**Comment by cbuto** [comment]
@mesosphere-mergebot backport release-2.2

**Pull request merged**